src/app/zap-templates/zcl/data-model/chip/mode-select-cluster.xml

```diff
@@ -27,7 +27,7 @@ limitations under the License.
     <cluster code="0x0050"/>
     <item name="Label" type="CHAR_STRING" optional="false" length="32"/>
```

Comment:
My copy of the spec says max length here is 64

Reply:
This is ignored (for the moment), but good catch.  Will fix.